src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,62 @@
+package duke.command;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+
+import duke.tasklist.TaskList;
+import duke.ui.UI;
+import duke.data.DataStorage;
+import duke.exception.DukeException;
+import duke.task.Task;
+
+/**
+ * Create add command class
+ */
+
+public class AddCommand extends Command {
+
+    private String type;
+    private LocalDate dueDate;
+    private LocalTime startTime;
+    private LocalTime endTime;
+
+    /**
+     * Constructor to create add command object
+     */
+    public AddCommand(String input, String type, LocalDate date, LocalTime startTime, LocalTime endTime) {
+        super(input);
+        this.type= type;
+        this.dueDate =date;
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+
+    /** Add task into task list and return task list
+     * @return TaskList
```

Comment:
Need to have empty line between description and parameter section or else the JavaDoc won't be rendered properly when generating documentation.